pkg/quarterdeck/db/models/users_test.go

```diff
@@ -494,7 +497,99 @@ func (m *modelTestSuite) TestUserSwitchOrganization() {
 	role, err = user.Role()
 	require.NoError(err, "Could not fetch role from user")
 	require.Equal("Member", role)
+}
+
+func (m *modelTestSuite) TestRemoveOrganization() {
```

Comment:
It feels like there are some other test cases that we may want to enumerate because this is one of those things we'll want _fully_ tested. Could we create a story to refactor this test into subtests and make sure we're testing the following cases: (this will follow the story about confirmation)

1. Cannot delete user who is owner if there is no other owner to the organization
2. Can delete a user in any role if they have no apikeys or project ownership (and another owner exists)
3. Cannot delete a user with apikeys and project ownership without token 
4. In an organization with 3 owners and projects/apikeys owned by 1 owner; deleting user with no projects/apikeys has no effect except to remove the user, deleting user with projects/apikeys and valid token clears out all projects/apikeys for the organization.
5. In an organization with 3 owners and projects/apikeys owned by 2. users; deleting a user with projects/apikeys and a valid token leaves the projects/apikeys owned by the other user.